src/routes/home/components/DisciplineCard.tsx

```diff
@@ -63,9 +63,12 @@ const DisciplineCard = ({
   }
 
   if (!discipline) {
+    if (error?.message !== NetworkError) {
```

Comment:
So still for invalid api key (e.g. revoked) there is just displayed nothing, leaving users wondering why there custom discipline just has disappeared.

You can test for this error, if you create an api key in test cms, add i to the app and revoke the api key in the test cms afterwards.

Reply:
Ah, I think I misunderstood your previous comment.

Instead of hiding the card on forbidden/unknown error, the card should be empty save for an error message and option to delete the card, same as in the manage selection screen?

Reply:
Oke, sorry about unclear previous comment.

Correct, on forbidden and unknown error, there should be displayed an error message and a button for deleting the custom discipline (remove api-key from storage).

Not sure how we want to handle unknown error for lunes standard discipline, I think here it would also make sense to add option/button to remove profession from selected professions

Reply:
> For network errors the arrow is still displayed in ManageSelection screen

Should the arrow be removed, or replaced by the red X? For Forbidden and Unknown the option to remove makes sense, not sure about NetworkError.

Reply:
Arrow should be removed and not be replace, because of the reasons you mentioned :+1:

Reply:
Done.

Wuick question, does the new DeletionModal component need a test suite? Since it's just a wrapper around Modal.